spotty/spotty/Utils/Extensions/UI/UIView+Animations.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  UIView+Animations.swift
+//  spotty
+//
+//  Created by Ivaylo Kalaydzhiev on 9.06.22.
+//
+
+import UIKit
+
+extension UIView {
+    
+    /// The UI View animates a spring bounce
+    func bounce() {
+        UIView.animate(withDuration: 1,
+                       delay: 0.0,
+                       usingSpringWithDamping: 0.2,
+                       initialSpringVelocity: 0.0,
+                       options: .allowUserInteraction,
+                       animations: {
+            self.bounds.size.width += 5.0
```

Comment:
In every animation, we will increase bounce sizes. Wouldn't it be better if we decrease the size after the animation is finished?

Reply:
Done. This method is not used anywhere yet, so I want to extract the magic numbers into the format which will make sense at that time.